### PR TITLE
Support for non variant models added

### PR DESCRIPTION
**Pull request by martinhoefling**
Non variant models support and test with non variant pulse.

### CODE REVIEW

**Comment by martinhoefling** [comment]
Looks like it was broken before. How do you prefer to resolve the issues?

**Comment by martinhoefling** [comment]
@jckuester ping

**Comment by martinhoefling** [comment]
golint and megacheck fixed

File: main.go [hunk 1]
```diff
@@ -108,7 +110,7 @@ func DecodedSignal(line string) (stop bool) {
 		}
 
 		switch device {
-		case GT_WT_01:
+		case GtWt01:
```
Comment (jckuester) [inline main.go]: I thought you wanted to add a new device for "GT-WT-01 variant", and not to overwrite the existing one?

Comment (martinhoefling) [reply main.go]: Just renamed it for the linter. The problem is, that the detection only uses the pulse length and not the actual values. Therefore, both are GT_WT_01 but fields are detected at runtime.

**Comment by martinhoefling** [comment]
closed in favor of #15 and #16

**Pull request closed without merging**